helix-core/src/syntax.rs

```diff
@@ -105,6 +105,8 @@ pub struct LanguageConfiguration {
     pub comment_token: Option<String>,
     pub text_width: Option<usize>,
     pub soft_wrap: Option<SoftWrap>,
+    #[serde(default)]
+    pub code_actions_on_save: HashSet<String>, // List of LSP code actions to be run in order upon saving
```

Comment:
A `HashSet` is ordered based on the item's hash value though. Let's just use a `Vec` here since I can also imagine usecases where you would want to execute action A, B then A again